shared/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "./tsconfig.paths.json",
   "compilerOptions": {
     "esModuleInterop": true,
-    "lib": ["ESNext"],
+    "lib": ["ESNext", "DOM"],
```

Comment:
What is this for?

Reply:
I had added that to use the `File` type defined by React. I don't think I need this any more.